### PR TITLE
Onboard Camel Quarkus scenario

### PR DESCRIPTION
**Pull request by oharan2**
Depends on: [#42415 ](https://github.com/openshift/release/pull/42415)

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

File: ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- **Camel-Quarkus official test suite repository**: [camel-q/camel-q-test-suite](https://gitlab.cee.redhat.com/jboss-fuse-qe/camel-q/camel-q-test-suite)
+
+## Purpose
+
+During the run CI takes the Dockerfile from `camel-quarkus-openshift-interop/openshift-ci` folder, builds an image from it and runs tests from [Camel Quarkus TS](https://gitlab.cee.redhat.com/jboss-fuse-qe/camel-q/camel-q-test-suite) modules defined by `$PROJECTS` variable.
+
+###### -TODO Slack channel
+Results are posted into [quarkus-qe](https://redhat-internal.slack.com/archives/C05CMLUAWTT) channel in Slack and new issue in [CEQ Jira](https://issues.redhat.com/projects/CEQ/summary) is created automatically for every failure. 
+
+Job history can be accessed from the [Prow Dashboard](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-quarkus-qe-quarkus-test-suite-main-quarkus-ocp4.14-lp-interop-quarkus-interop-aws) or via [dashboard](https://testgrid.k8s.io/redhat-openshift-lp-interop-release-4.14-informing#periodic-ci-quarkus-qe-quarkus-test-suite-main-quarkus-ocp4.14-lp-interop-quarkus-interop-aws&width=90).
```
[the remark this answers is not in the document]
Comment (oharan2) [reply ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/README.md]: Fixed

File: ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Repository for interop testings**: [jboss-fuse/camel-quarkus-openshift-interop](https://github.com/jboss-fuse/camel-quarkus-openshift-interop/tree/main)
+- **Camel-Quarkus official test suite repository**: [camel-q/camel-q-test-suite](https://gitlab.cee.redhat.com/jboss-fuse-qe/camel-q/camel-q-test-suite)
+
+## Purpose
+
+During the run CI takes the Dockerfile from `camel-quarkus-openshift-interop/openshift-ci` folder, builds an image from it and runs tests from [Camel Quarkus TS](https://gitlab.cee.redhat.com/jboss-fuse-qe/camel-q/camel-q-test-suite) modules defined by `$PROJECTS` variable.
+
+###### -TODO Slack channel
+Results are posted into [quarkus-qe](https://redhat-internal.slack.com/archives/C05CMLUAWTT) channel in Slack and new issue in [CEQ Jira](https://issues.redhat.com/projects/CEQ/summary) is created automatically for every failure.
```
[the remark this answers is not in the document]
Comment (oharan2) [reply ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/README.md:13]: @llowinge I'ts the channel I mentioned you that is archived. The correct one should be provided to us and i'll resolve this TODO. thanks.

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

File: ci-operator/step-registry/camel-quarkus/execute-tests/camel-quarkus-execute-tests-commands.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+shopt -s nullglob
+
+echo "Logging to oc:"
+sh ./oc_login.sh
+echo "Executing tests:"
+sh ./run.sh
+
+# Collect test reports into $ARTIFACT_DIR
+PROJECTS=$PROJECTS
```
Comment (chetna14manku) [inline ci-operator/step-registry/camel-quarkus/execute-tests/camel-quarkus-execute-tests-commands.sh:15]: what is the need of assigning PROJECTS value to same variable here? It should work without reassigning it.

Comment (oharan2) [reply ci-operator/step-registry/camel-quarkus/execute-tests/camel-quarkus-execute-tests-commands.sh:15]: Without the assignment shellcheck fails.

Comment (oharan2) [reply ci-operator/step-registry/camel-quarkus/execute-tests/camel-quarkus-execute-tests-commands.sh:15]: I've tried

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-quarkus-openshift-interop-main-camel-quarkus-ocp4.14-lp-interop-camel-quarkus-interop-aws

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**